src/EnergyPlus/api/runtime.py

```diff
@@ -118,7 +121,10 @@ def run_energyplus(self, state: c_void_p, command_line_args: List[Union[str, byt
         cli_args = cli_arg_type(*args_with_program_name)
         return self.api.energyplus(state, len(args_with_program_name), cli_args)
 
-    def issue_warning(self, message: Union[str, bytes]) -> None:
+    # def stop_simulation(self, state: c_void_p) -> None:
```

Comment:
Keep this around?

Reply:
I should probably keep it and just uncomment it, and wire it up to the underlying C function.  @kbenne 's PR will have some conflicts once this goes in, and while I'm resolving those, I'll uncomment this.  Good eye, though.

Reply:
There's also a couple of others in ```runtime.cc```. FYI

https://github.com/NREL/EnergyPlus/blob/5bf36f34bad6254a899b749d0c76a9d072c25124/src/EnergyPlus/api/runtime.cc#L269-L275

Reply:
Another keen eye. Those are a little different.  The way the User-Defined components work is by taking the name of a program from the IDF and running it explicitly.  This didn't fit well with the callback approach we have for all the other calling points, so these aren't currently enabled.

We should take time after 9.4 to think about how to do component models really well.  For now these will just sit here as commented placeholders.